### PR TITLE
Connecting React to Server

### PR DESCRIPTION
**Pull request by kerraspring**
- Created simple API call to server.js from client App.js
- In the end we would have our server script start both node and React,  then React handles all the rendering client side
- Server only handles data between client, server, DB, not rendering files.

### CODE REVIEW

**Review by codewithfan — COMMENTED** [review]
Awesome! Thank you for the help!

**Comment by sloth30799** [comment]
How did you make proxy work? I can't make it work no matter what I try....it kept returning to localhost 3000

**Comment by kerraspring** [comment]
React will run and open on 3000, the back end on 4000. You have to start
both for it to work

File: client/src/Components/Signup.jsx [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
+
 
 const Signup = () => {
+	const googleLogin = () => {
+		window.open("http://localhost:4000/auth/google", "_self");
+	};
+
 	return (
 		<div>
 			<form action='#'>
```
Comment (TheDariaEdits) [inline client/src/Components/Signup.jsx:13]: I'm confused by this portion. I thought we were using Google o-auth to login? If so why do we need signup form?

File: server/server.js [hunk 2]
```diff
@@ -9,6 +9,8 @@ const flash = require("express-flash");
 const cors = require("cors");
 const connectDB = require("./config/database");
 const authRoutes = require("./routes/auth")
+const mainRoutes = require("./routes/main");
+const PORT = process.env.PORT || 4000;
```
Comment (sloth30799) [inline server/server.js:13]: process.env has to been after dotenv require

File: client/src/Components/Signup.jsx (hunk 1 above)
Comment (sloth30799) [reply client/src/Components/Signup.jsx:13]: we don't need it to sign up but we can add it if we want to.

**Pull request merged**